src/openapi.yaml

```diff
@@ -632,6 +662,38 @@ paths:
     $ref: './resources/AccountRewardClaim.yaml'
   /admin/userSignUp:
     $ref: './resources/UserSignUp.yaml'
+  /environment/{environmentId}/job:
+    $ref: './resources/job/EnvironmentJob.yaml'
+  /environment/{environmentId}/job/status:
+    $ref: './resources/job/EnvironmentJobStatus.yaml'
+  /job/{jobId}:
+    $ref: './resources/job/Job.yaml'
+  /job/{jobId}/advancedSettings:
+    $ref: './resources/job/JobAdvancedSettings.yaml'
+  /job/{jobId}/deploy:
+    $ref: './resources/job/JobDeploy.yaml'
+  /job/{jobId}/restart:
+    $ref: './resources/job/JobRestart.yaml'
+  /job/{jobId}/stop:
+    $ref: './resources/job/JobStop.yaml'
+  /job/{jobId}/status:
+    $ref: './resources/job/JobStatus.yaml'
+  /job/{jobId}/deploymentHistory:
+    $ref: './resources/job/JobDeploymentHistory.yaml'
+  /job/{jobId}/instance:
+    $ref: './resources/job/JobCurrentInstance.yaml'
+  #/job/{jobId}/environmentVariable:
+  #/job/{jobId}/environmentVariable/{environmentVariableId}/override:
+  #/job/{jobId}/secret/{secretId}/override:
+  #/job/{jobId}/environmentVariable/{environmentVariableId}/alias:
+  #/job/{jobId}/secret/{secretId}/alias:
+  #/job/{jobId}/environmentVariable/{environmentVariableId}:
+  #/job/{jobId}/environmentVariable/{variableId}:
+  #/job/{jobId}/secret:
+  #/job/{jobId}/secret/{secretId}:
+  #/job/{jobId}/environmentVariable/import:
+  #/organization/{organizationId}/job/preview:
+  #/organization/{organizationId}/job/deploy:
```

Comment:
We don't expose those endpoints yet ?

Reply:
Because I need to make the schemas/resources :see_no_evil: and they are not yet there